.github/workflows/reusable-android-build.yml

```diff
@@ -129,7 +129,7 @@ jobs:
             echo "Version Code: $VERSION_CODE"
             echo "Version: $VERSION"
             # This will need to change currently looking for specific version and not place holder
-            sed -i -e "s/16022/$VERSION_CODE/g" -e "s/0.16.22/$VERSION/g" ./android/app/build.gradle 
```

Comment:
@jfmcquade Should this be added to the action `yarn scripts version`? This is what I normally use to bump the version but it only updates the other two places...

Reply:
Good point. Really we want to action #2136, but this is probably a good idea in the short term, I'll have a look

Reply:
If we're updating the android build number for content repos shouldn't we be using a variable coming from the content repo itself? E.g. the content repo version number...